src/test/run-make/debug-assertions/Makefile

```diff
@@ -0,0 +1,19 @@
+-include ../tools.mk
+
+all:
+	$(RUSTC) debug.rs -C debug-assertions=false
+	$(call RUN,debug)
+	$(RUSTC) debug.rs -C opt-level=1
```

Comment:
How about `-C opt-level=0` too (both with and without an explicit `debug-assertions`)?